pnpm-workspace.yaml

```diff
@@ -7,5 +7,6 @@ packages:
   - infra
   - test
   - ui
-  - merkle-tree-wasm/*
+  - merkle-tree-wasm
```

Comment:
with `merkle-tree-wasm/*` the package wasn't found when I filter it with: `pnpm --filter @anonklub/merkle-tree compile`